docs/playbook.md

```diff
@@ -36,6 +36,18 @@ Command to enable the flag:
 
 Refer to know more about CGO: https://www.golinuxcloud.com/cgo-tutorial/
 
+### Mount gets stuck with error: DefaultTokenSource: google: could not find default credentials.
+This state comes when we don't provide key-file as a mounting argument. In this
+case GCSFuse tries to fetch the default application credential, but got stuck
+due to absense of application-credentials on the VM.
+
+Resolve this issue by fetching the creds with the below command:
+Run "gcloud auth application-default login" command to fetch default creds
```

Comment:
you already mentioned
resolve this issue by fetching creds with below command
why do you need to say run x command again. just give the command.

Reply:
That was a mistake, forgot to delete the above line. Thanks!